property-services/src/main/resources/db/migration/main/V20191121135040__eg_pt_property_ddl.sql

```diff
@@ -0,0 +1,199 @@
+DROP TABLE IF EXISTS eg_pt_document;
+DROP TABLE IF EXISTS eg_pt_address;
+DROP TABLE IF EXISTS eg_pt_owner;
+DROP TABLE IF EXISTS eg_pt_institution;
+DROP TABLE IF EXISTS eg_pt_property;
+DROP TABLE IF EXISTS eg_pt_property_audit;
+DROP TABLE IF EXISTS eg_pt_address_audit;
+--> Property table
+
+CREATE TABLE eg_pt_property (
+
+   id                   CHARACTER VARYING (128) NOT NULL,
+   propertyId           CHARACTER VARYING (128),
+   tenantId             CHARACTER VARYING (256) NOT NULL,
+   accountId            CHARACTER VARYING (128) NOT NULL,
+   oldPropertyId        CHARACTER VARYING (128),
+   status               CHARACTER VARYING (128) NOT NULL,
```

Comment:
Do we not have search by status or other fields ? We'll have to indices on all searchable fields